src/algebra/order_functions.lean

```diff
@@ -12,6 +12,55 @@ variables {α : Type u} {β : Type v}
 
 attribute [simp] max_eq_left max_eq_right min_eq_left min_eq_right
 
+/-- A function `f` between preorders is strictly monotone if `a < b` implies `f a < f b`. -/
+def strict_mono [preorder α] [preorder β] (f : α → β) : Prop :=
```

Comment:
could those assumptions be weakened to `has_lt`?

Reply:
Done.